library/src/main/java/com/novoda/downloadmanager/lib/NotificationDisplayer.java

```diff
@@ -0,0 +1,314 @@
+package com.novoda.downloadmanager.lib;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.ContentUris;
+import android.content.Context;
+import android.content.Intent;
+import android.content.res.Resources;
+import android.graphics.Bitmap;
+import android.net.Uri;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.util.LongSparseArray;
+import android.support.v4.util.SimpleArrayMap;
+import android.text.TextUtils;
+import android.text.format.DateUtils;
+
+import com.novoda.downloadmanager.R;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+class NotificationDisplayer {
```

Comment:
No internal refactoring has done in this class, that could be done in future PRs